Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -870,7 +907,10 @@ public void RebuildObservers(bool initialize)
                 if (!newObservers.Contains(conn))
                 {
                     // removed observer
-                    conn.RemoveFromVisList(this, false);
+                    if (conn.playerController != null)
+                    {
+                        conn.playerController.RemoveFromVisList(this, false);
```

Comment:
```suggestion
                        conn.playerController?.RemoveFromVisList(this, false);
```

Reply:
using this unity components gives warnings. apparently we are supposed to null check.